provider-integration/im2/docs/src/slurm/id-management.md

```diff
@@ -647,7 +708,7 @@ registered before, then the script __must:__
 ```json
 {
     /* string */
-    "ucloudProjectId": "",
+    "UCloudProjectId": "",
```

Comment:
If that is needed, then I think we have a bug in the code rather than the configuration.